spec/rails_helper.rb

```diff
@@ -78,5 +81,14 @@
 VCR.configure do |config|
   config.cassette_library_dir = 'spec/fixtures/vcr_cassettes'
   config.hook_into :webmock
+  config.configure_rspec_metadata!
```

Comment:
I'm assuming this is where you were trying to get VCR to start creating cassettes? I'm not quite sure if I'm right, but just flagging my slack message from 3/26 about VCR with this particular portion of our project:

"I haven't had it make any cassettes at all for me, but I also think we don't necessarily need it to/need to use VCR, at least for the things I've done so far. Since we're only calling a local API that we're responsible for, we don't have to worry about rate limiting (or even internet access) and since we're using factorybot for test data, we don't really need the seed data for anything other than the postman testing suite"